src/Effects/Effects.UWP/Effects/Entry/EntryRemoveBorder.cs

```diff
@@ -2,33 +2,31 @@
 using FormsCommunityToolkit.Effects.UWP;
 using Xamarin.Forms;
 using Xamarin.Forms.Platform.UWP;
-using Thickness = Windows.UI.Xaml.Thickness;
-
-[assembly: ExportEffect(typeof(RemoveBorderEffect), nameof(RemoveBorderEffect))]
 
+[assembly: ExportEffect(typeof(EntryRemoveBorder), nameof(EntryRemoveBorder))]
 namespace FormsCommunityToolkit.Effects.UWP
 {
-    public class RemoveBorderEffect : PlatformEffect
+    public class EntryRemoveBorder : PlatformEffect
     {
-        Thickness old;
+        private Windows.UI.Xaml.Thickness _old;
```

Comment:
Also here, we need the Thickness of UWP not the one from Xamarin forms, so full type name needed